.github/workflows/nightly-tests.yml

```diff
@@ -60,13 +60,13 @@ jobs:
           target
         key: ${{ runner.os }}-cargo-release-${{ hashFiles('**/Cargo.toml') }}
     - name: Install Rust toolchain 1.85
-      run: rustup toolchain install 1.85-x86_64-unknown-linux-gnu
+      run: rustup toolchain install 1.85
     - name: Install nightly
-      run: rustup toolchain install nightly-2025-05-14-x86_64-unknown-linux-gnu
-    - name: Install std source
-      run: rustup component add rust-src --toolchain nightly-2025-05-14-x86_64-unknown-linux-gnu
+      run: rustup toolchain install nightly-2025-05-14 --component rust-src
+    - name: Install nightly
+      run: rustup toolchain install nightly-2025-02-14 --component rust-src
```

Comment:
This is the only change required to fix nightly (hopefully), the rest is just simplifications